rct229/rules/section21/section21rule18.py

```diff
@@ -0,0 +1,66 @@
+from rct229.data.schema_enums import schema_enums
+from rct229.rule_engine.rule_base import RuleDefinitionBase
+from rct229.rule_engine.rule_list_indexed_base import RuleDefinitionListIndexedBase
+from rct229.rule_engine.user_baseline_proposed_vals import UserBaselineProposedVals
+
+APPLICABLE_SYS_TYPES = [
+    "SYS-1",
+    "SYS-5",
+    "SYS-7",
+    "SYS-11.2",
+    "SYS-12",
+    "SYS-1A",
+    "SYS-7A",
+    "SYS-11.2A",
+    "SYS-12A",
+]
+
+FUEL_SOURCE_OPTION = schema_enums["EnergySourceOptions"]
```

Comment:
I think we've been not including the work "OPTION" in the names of our enumeration constants.  Then it is used as `FUEL_SOURCE. NATURAL_GAS`, which seems natural (no pun intended).

Reply:
It totally makes sense to me and this one was fixed.